cocos/spine/attach-util.ts

```diff
@@ -32,44 +32,54 @@ import { Node } from '../scene-graph';
 const tempMat4 = new Mat4();
 
 /**
- * @en Attach node tool
- * @zh 挂点工具类
+ * @en Attach node tool.
+ * @zh 挂点工具类。
  * @class sp.AttachUtil
  */
 export class AttachUtil {
     protected _inited = false;
-    protected _skeleton:spine.Skeleton|null = null;
-    protected _skeletonNode:Node|null = null;
-    protected _skeletonComp:Skeleton|null = null;
+    protected _skeleton: spine.Skeleton|null = null;
+    protected _skeletonNode: Node|null = null;
+    protected _skeletonComp: Skeleton|null = null;
 
     constructor () {
         this._inited = false;
         this._skeleton = null;
         this._skeletonNode = null;
         this._skeletonComp = null;
     }
-
-    init (skeletonComp:Skeleton) {
+    /**
+     * @en Initialization.
+     * @zh 初始化。
+     * @param skeletonComp @en skeleton object @zh skeleton组件
```

Comment:
Please add some description about why need initialization and why need pass in a skeleton component